mbuild/examples/alkane/alkane.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, n=3, cap_front=True, cap_end=True):
             n += 1
         if not cap_end:
             n += 1
-        chain = mb.Polymer(CH2(), n=n-2, port_labels=('up', 'down'))
+        chain = mb.my_recipes.Polymer(CH2(), n=n-2, port_labels=('up', 'down'))
```

Comment:
foo

Reply:
I am going to resolve this comment since it was used to test permissions 😉